tools/rostest/src/rostest/rostest_main.py

```diff
@@ -98,6 +98,9 @@ def rostestmain():
     parser.add_option("--results-base-dir", metavar="RESULTS_BASE_DIR",
                       help="The base directory of the test results. The test result file is " +
                            "created in a subfolder name PKG_DIR.")
+    parser.add_option("--port", metavar="PORT",
+                      dest="port", default=None,
+                      help="ROS Master port number")
```

Comment:
Please update the help text to something like this:

    ROS Master port number (default: random unused port)